server/src/crawl.js

```diff
@@ -40,10 +42,25 @@ let watch = async () => {
 
             if (minBlockCrawl > parseInt(setting.meta_value)) {
                 setting.meta_value = minBlockCrawl
+                isSend = true
                 await setting.save()
             }
 
             if (String(maxBlockNum) === String(minBlockCrawl)) {
+                const a = await web3.eth.getBlock(minBlockCrawl)
```

Comment:
we do not need to getBlock.
Just calculate sleeping time  = X * 0.5
If Sleeping time > 2 minutes -> notify